content/guides/hosting/hosting-options/self-managed/kubernetes-operator/_index.md

```diff
@@ -756,6 +798,32 @@ global:
     -----END CERTIFICATE-----
 ```
 
+CA certificates can also be stored in a ConfigMap:
+```yaml
+global:
+  caCertsConfigMap: custom-ca-certs
+```
+
+The ConfigMap must look like this:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: custom-ca-certs
+data:
+  ca-cert1.crt: |
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+  ca-cert2.crt: |
+    -----BEGIN CERTIFICATE-----
+    ...
+    -----END CERTIFICATE-----
+```
+
+**Important:**
```

Comment:
Also, this is no longer the righrt way to do an admonition . Instead, use the Hugo shortcode:
```
{{% alert %}}
...
{{% /alert %}}
```